docs/guides/integration/dependency-bots.md

```diff
@@ -0,0 +1,25 @@
+# Dependency bots
+
+## Renovate
+
+uv is supported by [Renovate](https://github.com/renovatebot/renovate).
+
+With `lockFileMaintenance` enabled, Renovate will automatically detect `uv.lock` and suggest updates
+to `uv.lock` files.
+
+```json5 title="renovate.json5"
+{
+  $schema: "https://docs.renovatebot.com/renovate-schema.json",
+  lockFileMaintenance: {
+    enabled: true,
+  },
+}
+```
```

Comment:
`lockFileMaintenance` is not required for uv support, it mostly allows refreshing transitive dependencies (or also direct dependencies if using range of versions for dependencies in `pyproject.toml`) in the lock file on a regular basis, so this should probably be a dedicated paragraph.

I'd probably go with something like:

```suggestion
Renovate will detect the presence of `uv.lock` file to determine that uv is used for managing
dependencies, and will suggest updates to
both [project dependencies](../../concepts/dependencies.md#project-dependencies), [optional dependencies](../../concepts/dependencies.md#optional-dependencies)
and [development dependencies](../../concepts/dependencies.md#development-dependencies), updating
`uv.lock` file along the way.

It is also possible to refresh `uv.lock` file on a regular basis (for instance to update transitive
dependencies) by
enabling [`lockFileMaintenance`](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance)
option:

```json5 title="renovate.json5"
{
  $schema: "https://docs.renovatebot.com/renovate-schema.json",
  lockFileMaintenance: {
    enabled: true,
  },
}
```
```

Reply:
Thanks!